template/android/fastlane/Fastfile

```diff
@@ -48,12 +56,7 @@ platform :android do
 
   desc "====CODE PUSH===="
   lane :codepush do |options|
-    appcenter_codepush(
-      env: options[:env],
-      owner_name: owner_name,
-      my_app: my_app,
-      platform: platform
-    )
+    sh "cd .. && cd .. && appcenter codepush release-react --token #{ENV["APPCENTER_TOKEN_ANDROID"]} -a #{owner_name}/#{my_app} -d #{options[:env]} -t #{version} --mandatory true"
```

Comment:
Is there any way to improve sh cd ex: cd ../.. instead of cd .. && cd ..

Reply:
updated